src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,11 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
     public String getRandomColor() {
-        return null;
+        int index = new Random().nextInt(Colors.values().length);
+        Colors color = Colors.values()[index];
+        return String.valueOf(color);
     }
```

Comment:
Instead of creating an instance of `Random` every time the method is called, consider defining it as a class-level variable. This will prevent unnecessary object creation and make the code more efficient.